.github/workflows/cypress.yml

```diff
@@ -43,6 +49,9 @@ jobs:
       - name: Start Cypress tests
```

Comment:
```suggestion
      - name: Start Cypress tests index ${{ matrix.ci_node_index }}
```

Something similar to distinguish it bit more nicely

Reply:
As a step name, it seems fine. The job name seems like adequate distinction. If insisting on including the index, I think something like "Start Cypress tests (0)" should be sufficient.

Reply:
Not insistent. Was a suggestion that might look bit more nicely when reading through steps, but am 💯  leaving as is